### ISSUE TITLE
Create three "content session" meetings

### ISSUE THREAD

**Issue by choldgraf** [2i2c-org/team-compass#518]
### Context

We are juggling a lot of different issues of varying degrees of concreteness and importance. Many of them require "shaping", or discussion to align on a path forward and responsibility for completing them. We currently do this kind of work purely asynchronously, or as part of one-on-ones between team members.

We've discussed the need to create more bridges between the various teams of 2i2c, and so I wonder if we could create a new meeting type that facilitates these conversations.[^refs]

[^refs]: Related to:
    - https://github.com/2i2c-org/team-compass/issues/517

### Proposal

I propose that we define **content session** meetings each week. Here are a few goals of these meetings:

- Discuss and shape the most important issues within a functional area
- Plan our next actions, delegate, and set checkpoints
- Encourage discussion and collaboration across teams

They happen each week at the same time, and are no longer than 60 minutes. Each meeting is led by the leads of each group. Anybody is welcome to join, and the meeting leads are empowered to request attendance from others ahead of time.

The agenda is driven by the leads of each functional area who collaboratively define the most important topics to discuss at least 24 hours ahead of time. Anybody is welcome to attend but the leads are empowered to _request_ attendance from specific people if they need to be a part of the conversation.

To start, I suggest that we define three functional groups / meetings:

- Product and Engineering ([every Tuesday at **4pm CET**](https://www.worldtimebuddy.com/?qm=1&lid=683844,2988507,2643743,100,6167865,5391959&h=2988507&date=2022-9-13&sln=16-17&hf=undefined&c=1507))
- Community and Partnerships ([every Tuesday at **4pm CET**](https://www.worldtimebuddy.com/?qm=1&lid=683844,2988507,2643743,100,6167865,5391959&h=2988507&date=2022-9-15&sln=16-17&hf=undefined&c=1507))
- Organizational strategy ([every other Wednesday at **5pm CET**](https://www.worldtimebuddy.com/?qm=1&lid=683844,2988507,2643743,100,6167865,5391959&h=2988507&date=2022-9-14&sln=17-18&hf=undefined&c=1773))

### References

- Part of: https://github.com/2i2c-org/team-compass/issues/496

**Comment by sgibson91** [comment]
> * Product and Engineering ([every Tuesday at **4pm CET**](https://www.worldtimebuddy.com/?qm=1&lid=683844,2988507,2643743,100,6167865,5391959&h=2988507&date=2022-9-13&sln=16-17&hf=undefined&c=1507))
> * Community and Partnerships ([every Tuesday at **4pm CET**](https://www.worldtimebuddy.com/?qm=1&lid=683844,2988507,2643743,100,6167865,5391959&h=2988507&date=2022-9-15&sln=16-17&hf=undefined&c=1507))

Did you mean "Thursday" for the second one?

**Comment by choldgraf** [comment]
Yes! Sorry (and those dates/times are just proposals that are loosely based on when one-on-ones are already happening, they can be adjusted)

**Comment by sgibson91** [comment]
Of course! Just wanted to make sure you weren't intentionally proposing a clash :D

**Comment by damianavila** [comment]
In general, I like the idea, although it is not clear in my head yet how this content session meeting interplays with cycle/sprint meeting... 🤔.
